holobot.go

```diff
@@ -566,12 +566,15 @@ func HandleReactions(event *model.WebSocketEvent) (err error) {
 func HandleSourceRequests(event *model.WebSocketEvent) (err error) {
 	reaction := model.ReactionFromJson(strings.NewReader(event.Data["reaction"].(string)))
 	post, _ := client.GetPost(reaction.PostId, "")
+	channel, _ := client.GetChannel(post.ChannelId, "")
+	team, _ := client.GetTeam(channel.TeamId, "")
 	postuser, _ := client.GetUser(post.UserId, "")
 	reactuser, _ := client.GetUser(reaction.UserId, "")
+	messagesrc := strings.Replace(post.Message, "\n", "\n    ", -1)
 	// if you react with :u55b6:
 	if reaction.EmojiName == "u55b6" {
 		SendMsgToDebuggingChannel(fmt.Sprintf("**Source request reaction detected!!**\n**Event data:**%v", event.Data), "")
-		SendDirectMessage(reactuser.Id, "Here's plaintext of @"+postuser.Username+"'s message:\n```\n"+post.Message+"\n```")
+		SendDirectMessage(reactuser.Id, "Here's plaintext of @"+postuser.Username+"'s [message](http://"+config.Domain+"/"+team.Name+"/pl/"+post.Id+"):\n\n    "+messagesrc)
```

Comment:
This is all fine.  But as a style thing, once there's lots of + things, I like to shift to `fmt.Sprintf` because it's easier to read and edit the strings...